hack/install-kwok.sh

```diff
@@ -0,0 +1,102 @@
+set -euo pipefail
```

Comment:
Top-level script comment that describes script usage, what the script is intended to do, and script options/flags?

Reply:
Can do